contracts/core/libs/attestations-registry/AttestationsRegistryState.sol

```diff
@@ -10,13 +10,19 @@ contract AttestationsRegistryState {
   bool internal _initializing;
   bool internal _paused;
   address internal _owner;
+  // Storing the tag name for each tags index
+  mapping(uint256 => bytes32) internal _tags;
```

Comment:
tagIndex is `uint8` but the mapping maps an `uint256` to the name of the tag. Shouldn't it be 
```solidity
mapping(uint8 => bytes32) internal _tags;
```